.github/workflows/build-and-push-image.yml

```diff
@@ -115,8 +115,8 @@ jobs:
       - name: Npm install
         run: npm install
 
-      - name: Run cypress
-        run: npx cypress run --env url='${{ secrets.AZURE_ENDPOINT }}',authorizationHeader='${{ secrets.AUTH_HEADER_CYPRESS }}'
+      - name: Run cypress (${{ needs.set-env.outputs.environment }})
+        run: npm run cy:run -- --env url='${{ secrets.AZURE_ENDPOINT }}',authorizationHeader='${{ secrets.AUTH_HEADER_CYPRESS }}, academisationApiUrl=${{secrets.CYPRESS_ACADEMISATION_API_URL}}, academisationApiKey=${{ secrets.CYPRESS_ACADEMISATION_API_KEY}}'
```

Comment:
Are these extra variables/secrets needed? I can't find any reference to them in the Cypress code

Reply:
Ah I meant the academisationApiUrl and academisationApiKey, I can't see references to them being used by the Cypress tests

Reply:
@DrizzlyOwl had created those for me, I have them on my local, I will send them to you shortly. but just incase we do use them in the future at least they are correct. I can update the readme file about those.